website/docs/validating-admission-policy.md

```diff
@@ -40,6 +40,10 @@ In summary, these are potential options when running Gatekeeper:
 
 Find out more about different [enforcement points](enforcement-points.md)
 
+:::note
+CEL validation is in beta and `--experimental-enable-k8s-native-validation` is turned on by default.
```

Comment:
do we want to say something like "CEL policy validation through Gatekeeper and VAP generation is in ..."